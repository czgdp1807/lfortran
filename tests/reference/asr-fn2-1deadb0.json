{
    "basename": "asr-fn2-1deadb0",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fn2.f90",
    "infile_hash": "9f61561c47397aa03014fe07094ac838ada4865e513025e384edae00",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fn2-1deadb0.stdout",
    "stdout_hash": "e47377a2319f58cd1e55694022a86927ed8a08c04f70647c18871cf3",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}