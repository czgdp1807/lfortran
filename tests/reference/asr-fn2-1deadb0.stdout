(TranslationUnit
    (SymbolTable
        1
        {
            a:
                (Function
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    a
                    (FunctionType
                        []
                        (Integer 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 2 a)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                ),
            b:
                (Function
                    (SymbolTable
                        3
                        {
                            b:
                                (Variable
                                    3
                                    b
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    b
                    (FunctionType
                        []
                        (Real 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 3 b)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                ),
            c:
                (Function
                    (SymbolTable
                        4
                        {
                            c:
                                (Variable
                                    4
                                    c
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Logical 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    c
                    (FunctionType
                        []
                        (Logical 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 4 c)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                ),
            d:
                (Function
                    (SymbolTable
                        5
                        {
                            r:
                                (Variable
                                    5
                                    r
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    d
                    (FunctionType
                        []
                        (Integer 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 5 r)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                ),
            e:
                (Function
                    (SymbolTable
                        6
                        {
                            r:
                                (Variable
                                    6
                                    r
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    e
                    (FunctionType
                        []
                        (Real 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 6 r)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                ),
            f:
                (Function
                    (SymbolTable
                        7
                        {
                            r:
                                (Variable
                                    7
                                    r
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Logical 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    f
                    (FunctionType
                        []
                        (Logical 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    []
                    []
                    (Var 7 r)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                )
        })
    []
)
